sentinel2download/downloader.py

```diff
@@ -18,10 +18,16 @@
 PRODUCT_TYPE = namedtuple('type', 'L2A L1C')('L2A', 'L1C')
 
 BANDS = frozenset(('TCI', 'B01', 'B02', 'B03', 'B04', 'B05', 'B06',
-                   'B07', 'B08', 'B8A', 'B09', 'B10', 'B11', 'B12',))
+                   'B07', 'B08', 'B8A', 'B09', 'B10', 'B11', 'B12', 'CLD'))
 
 CONSTRAINTS = MappingProxyType({'CLOUDY_PIXEL_PERCENTAGE': 100.0, 'NODATA_PIXEL_PERCENTAGE': 100.0, })
 
+FOLDER_SUFFIX = "_$folder$"
+
+
+def is_dir(blob):
```

Comment:
Please, place the function in class as staticmethod. Blobs are entire objects.

Reply:
Done:
https://github.com/QuantuMobileSoftware/sentinel2tools/pull/7/commits/9a4db75f0d770bc2f9170d94d20037dbfcd110d9